e2e-identity/Cargo.toml

```diff
@@ -10,6 +10,9 @@ publish = false
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[lints.rust]
+unexpected_cfgs = { level = "allow", check-cfg = ['cfg(ci)'] }
+
```

Comment:
I think this should rather be addressed in `e2e-identity/build.rs`, where we tell rustc about `ci`.
See https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-check-cfg.